evap/evaluation/models.py

```diff
@@ -348,6 +348,15 @@ def can_user_see_results(self, user):
             return self.can_user_see_course(user)
         return False
 
+    def can_user_see_grades(self, user):
+        if user.is_reviewer:
+            return True
+        if self.state == 'published':
```

Comment:
maybe we should change the order and do a  `if self.state != 'published': return False`